package.json

```diff
@@ -995,6 +989,42 @@
                 "when": "editorTextFocus && neovim.init && neovim.mode == insert && neovim.ctrlKeysInsert.delete && editorLangId not in neovim.editorLangIdExclusions",
                 "args": "<C-Del>"
             },
+            {
+                "command": "vscode-neovim.send-cmdline",
+                "key": "tab",
+                "when": "neovim.init && neovim.mode == cmdline",
+                "args": "<tab>"
+            },
+            {
+                "command": "vscode-neovim.send-cmdline",
+                "key": "shift+tab",
+                "when": "neovim.init && neovim.mode == cmdline",
+                "args": "<S-tab>"
+            },
+            {
+                "command": "vscode-neovim.send-cmdline",
+                "key": "down",
+                "when": "neovim.init && neovim.mode == cmdline",
+                "args": "<down>"
+            },
+            {
+                "command": "vscode-neovim.send-cmdline",
+                "key": "up",
+                "when": "neovim.init && neovim.mode == cmdline",
+                "args": "<up>"
+            },
```

Comment:
Huh, I would have thought this change would have allowed for up/down to go through command history, but it doesn't on my build. Is that expected?

Reply:
Yes, up/down works for me, so does C-n/C-p for navigating history.

Reply:
Hm... I wonder why it doesn't work for me.


https://github.com/vscode-neovim/vscode-neovim/assets/977151/c8b33c22-46df-436e-9689-b9d318d11628

Reply:
Do you maybe have conflicting bindings in your vscode? 

Actually, mapping the arrow keys in the quickpick might only currently be supported in vscode-insiders. It will begin to work on a later vscode update. https://github.com/microsoft/vscode/pull/211756

Reply:
Yeah, that PR has not been released yet.

Reply:
ah yep, I'm not on Insiders

> no but it will be released for the May iteration. Likely released first week of June.

Sounds good :)